pkg/tools/gen/testdata/jsonschema/complex-workflow/expect.k

```diff
@@ -0,0 +1,1747 @@
+    issue_comment?: WorkflowOnOneOf2IssueCommentOneOf0 | str | WorkflowOnOneOf2IssueComment
+    issues?: WorkflowOnOneOf2IssuesOneOf0 | str | WorkflowOnOneOf2Issues
+    label?: WorkflowOnOneOf2LabelOneOf0 | str | WorkflowOnOneOf2Label
+    member?: WorkflowOnOneOf2MemberOneOf0 | str | WorkflowOnOneOf2Member
+    milestone?: WorkflowOnOneOf2MilestoneOneOf0 | str | WorkflowOnOneOf2Milestone
+    page_build?: WorkflowOnOneOf2PageBuildOneOf0 | str
+    project?: WorkflowOnOneOf2ProjectOneOf0 | str | WorkflowOnOneOf2Project
+    project_card?: WorkflowOnOneOf2ProjectCardOneOf0 | str | WorkflowOnOneOf2ProjectCard
+    project_column?: WorkflowOnOneOf2ProjectColumnOneOf0 | str | WorkflowOnOneOf2ProjectColumn
+    public?: WorkflowOnOneOf2PublicOneOf0 | str
+    pull_request?: WorkflowOnOneOf2PullRequestOneOf0 | str | WorkflowOnOneOf2PullRequest
+    pull_request_review?: WorkflowOnOneOf2PullRequestReviewOneOf0 | str | WorkflowOnOneOf2PullRequestReview
+    pull_request_review_comment?: WorkflowOnOneOf2PullRequestReviewCommentOneOf0 | str | WorkflowOnOneOf2PullRequestReviewComment
+    pull_request_target?: WorkflowOnOneOf2PullRequestTargetOneOf0 | str | WorkflowOnOneOf2PullRequestTarget
+    push?: WorkflowOnOneOf2PushOneOf0 | str | WorkflowOnOneOf2Push
+    registry_package?: WorkflowOnOneOf2RegistryPackageOneOf0 | str | WorkflowOnOneOf2RegistryPackage
+    release?: WorkflowOnOneOf2ReleaseOneOf0 | str | WorkflowOnOneOf2Release
+    status?: WorkflowOnOneOf2StatusOneOf0 | str
+    watch?: WorkflowOnOneOf2WatchOneOf0 | str
+    workflow_call?: WorkflowOnOneOf2WorkflowCall
+    workflow_dispatch?: WorkflowOnOneOf2WorkflowDispatch
+    workflow_run?: WorkflowOnOneOf2WorkflowRunOneOf0 | str | WorkflowOnOneOf2WorkflowRun
+    repository_dispatch?: WorkflowOnOneOf2RepositoryDispatchOneOf0 | str
+    schedule?: [WorkflowOnOneOf2ScheduleItems0]
+
+    check:
+        len(schedule) >= 1
+
+schema WorkflowOnOneOf2BranchProtectionRule:
+    r"""
+    Runs your workflow anytime the branch_protection_rule event occurs. More than one activity type triggers this event.
+
+    Attributes
+    ----------
+    types : ["created" | "edited" | "deleted"], optional, default is ["created" "edited" "deleted"]
+        Selects the types of activity that will trigger a workflow run. Most GitHub events are triggered by more than one type of activity. For example, the event for the release resource is triggered when a release is published, unpublished, created, edited, deleted, or prereleased. The types keyword enables you to narrow down activity that causes the workflow to run. When only one activity type triggers a webhook event, the types keyword is unnecessary.
+        You can use an array of event types. For more information about each event and their activity types, see https://help.github.com/en/articles/events-that-trigger-workflows#webhook-events.
+    """
+
+    types?: ["created" | "edited" | "deleted"] = ["created" "edited" "deleted"]
```

Comment:
The default value `["created" "edited" "deleted"]` may have better format ["created", "edited", "deleted"].

Reply:
Done